release 0.68.0 (#3964)
SwissDataScienceCenter/renku: CHANGELOG.rst

Individual Components
~~~~~~~~~~~~~~~~~~~~~

- `renku-data-services 0.39.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.39.0>`_
- `renku-data-services 0.40.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.40.0>`_
- `renku-data-services 0.41.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.41.0>`_
- `renku-data-services 0.42.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.42.0>`_
- `renku-data-services 0.43.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.43.0>`_
- `renku-gateway 1.5.0 <https://github.com/SwissDataScienceCenter/renku-gateway/releases/tag/1.5.0>`_
- `renku-gateway 1.5.1 <https://github.com/SwissDataScienceCenter/renku-gateway/releases/tag/1.5.1>`_
- `renku-python 2.9.3 <https://github.com/SwissDataScienceCenter/renku-python/releases/tag/v2.9.3>`_
- `renku-python 2.9.4 <https://github.com/SwissDataScienceCenter/renku-python/releases/tag/v2.9.4>`_
- `amalthea-sessions 0.18.2 <https://github.com/SwissDataScienceCenter/amalthea/releases/tag/0.18.2>`_

> - `renku-ui 3.55.0 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.55.0>`_

Reviewer: For completeness:

```suggestion
- `renku-ui 3.54.0 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.54.0>`_
- `renku-ui 3.55.0 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.55.0>`_
```

Reviewer 2: I will accept this as soon as the renku-deploy in the CI pipeline finishes. Otherwise it will get stuck.

.. _changelog:

0.68.0
------

> This version of Renku introduces several new features. The most notable of these are: being able to link and access data from Zenodo, Dataverse and similar providers, setting environment variables in session launchers and being able to migrate legacy Renku projects from within the new Renku UI.

Reviewer 3: ```suggestion
This version of Renku introduces several new features. The most notable of these are: 

- using published datasets from Zenodo, Dataverse and similar providers as Data Connectors
- setting environment variables in session launchers
- migrating legacy Renku projects from within the new Renku UI

The release also includes a completely reworked launcher/session interface. 
```

User-Facing Changes
~~~~~~~~~~~~~~~~~~~

**🌟 New Features**

> - **UI**: Datasets published on platforms such as Zenodo or Dataverse can be linked to
>   a project by using their reference DOI (Digital Object Identifier).

Reviewer 3: ```suggestion
- **UI**: Datasets published on platforms such as Zenodo or Dataverse can be linked to
  a project as Data Connectors by using their reference DOI (Digital Object Identifier).
```

- `renku-data-services 0.41.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.41.0>`_
- `renku-data-services 0.42.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.42.0>`_
- `renku-data-services 0.43.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.43.0>`_

> - `renku-data-services 0.43.1 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.43.0>`_

Reviewer 4: ```suggestion
- `renku-data-services 0.43.1 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.43.1>`_
```
